### PR TITLE
Add OWD charter

### PR DESCRIPTION
**Pull request by Elchi3**
Initial draft of the OWD charter document.

### CODE REVIEW

**Review by dontcallmedom — APPROVED** [review]

**Review by jorydotcom — APPROVED** [review]

**Review by torgo — APPROVED** [review]
lgtm

**Review by robnyman — APPROVED** [review]

File: charter.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Core Values
+
+Open Web Docs believes open web platform documentation is critical digital infrastructure that benefits everyone. To demonstrate that belief, we: 
+
+- _focus_ on documentation that has a positive impact on end-users and web developers.
+- _prioritize_ based on needs of the global community of web developers and designers. 
+- _maintain_ an independent editorial voice, with a “firewall” between fiscal and editorial matters. 
+- _adhere_ to open source principles of transparency in project governance, investment, and feature development in order to maintain trustworthiness and credibility with the web community. 
+- _work_ to make the platform more equitable, investing in documenting internationalization, accessibility, privacy and diversity components of the web.
```
Comment (chrisdavidmills) [inline charter.md:16]: "...privacy and diversity components of the web" -> "...privacy, diversity, and other critical components of a healthy World Wide Web"...?

Comment (chrisdavidmills) [reply charter.md:16]: Also maybe not mention diversity here, as you cover it in the below bullet too?

File: charter.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Core Values
+
+Open Web Docs believes open web platform documentation is critical digital infrastructure that benefits everyone. To demonstrate that belief, we: 
+
+- _focus_ on documentation that has a positive impact on end-users and web developers.
+- _prioritize_ based on needs of the global community of web developers and designers. 
+- _maintain_ an independent editorial voice, with a “firewall” between fiscal and editorial matters. 
+- _adhere_ to open source principles of transparency in project governance, investment, and feature development in order to maintain trustworthiness and credibility with the web community. 
+- _work_ to make the platform more equitable, investing in documenting internationalization, accessibility, privacy and diversity components of the web. 
+- _emphasize_ diversity and inclusion across all our efforts, including project contributions, community organization, and resource allocation.
```
Comment (chrisdavidmills) [inline charter.md:17]: I'd remove the first comma as I think it will make the sentence read better.

File: charter.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## What We Do
+To execute our mission and values, we contribute to MDN and other related efforts with an independent editorial voice, rooted in the web technology landscape reflecting the lived experience of web developers. We take a lead role on specific content projects within these platforms, in coordination with these platforms where appropriate.
```
Comment (chrisdavidmills) [inline charter.md:20]: "MDN Web Docs and other related documentation platforms..."

Comment (chrisdavidmills) [reply charter.md:20]: "...rooted in the web technology landscape reflecting..." -> "...rooted in the web technology landscape and reflecting..."

Comment (chrisdavidmills) [reply charter.md:20]: "...within these platforms, in coordination with these platforms where appropriate." -> "...within these platforms, in coordination with their core teams where appropriate." ?

File: charter.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+We invest in and protect Open Web Documentation as a critical piece of infrastructure which other organizations can build upon and are incentivized to protect. We ensure developers have the best information available to make technology and design decisions and to implement on the web platform.
```
Comment (chrisdavidmills) [inline charter.md:22]: This reads slightly awkwardly, and is somewhat repetitious ("critical digital infrastructure" also mentioned in the "Core Values" section).

How about this:

"Open Web Docs works hard both to invest in and protect documentation for the open web platform, and to provide incentives and a platform for other related organizations to get involved in this work. We ensure developers have the best information available to make technology and design decisions and implement on the web platform."

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Some editorial review comments.

File: charter.md (hunk 4 above)
Comment (Elchi3) [reply charter.md]: Thanks, updated! I left out the "works hard" part of this, though :)

**Comment by Elchi3** [comment]
Thanks a lot, Chris! Editorial updates made.

**Review by chrisdavidmills — APPROVED** [review]
Looking good, cheers @Elchi3 !

**Review by adrianhopebailie — APPROVED** [review]

**Comment by Elchi3** [comment]
Thanks everyone! Charter ratified.

**Pull request merged**